### PR TITLE
1-4 done

### PR DESCRIPTION
**Pull request by ZuzanaAK**
I am not entirely sure if this is how it was supposed to look like

### CODE REVIEW

File: src/navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import 'bulma/css/bulma.css';
+import {CoolButton} from "../coolbutton/CoolButton"
+
+function Navbar() {
+
```
Comment (In-S4n3) [inline src/navbar/Navbar.js:6]: Here you have to change the class to className to not have errors on the console.

File: src/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                >
+                  <span class="icon">
+                    <i class="fas fa-download"></i>
+                  </span>
+                  <span>Download</span>
+                </a>
+              </p>
+
+              <p className="control">
+                <CoolButton name = "Login" isSmall isDanger className="is-rounded my-class"></CoolButton>
```
Comment (In-S4n3) [inline src/navbar/Navbar.js:120]: Check my comment bellow to fix this after correcting the CoolButton component.

File: src/App.js [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+import {FormField} from "./formfield/FormField"
+import {CoolButton} from "./coolButton/CoolButton"
+import SignUp from "./signup/Signup"
 
 const App = () => {
-  return <h1>Bulma</h1>;
+  return (
+    <React.Fragment>
+      {/* <Navbar /> 
+
```
Comment (In-S4n3) [inline src/App.js:12]: Uncomment before you pullrequest :p

File: src/App.js [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+  return (
+    <React.Fragment>
+      {/* <Navbar /> 
+    
+      <FormField label="Name" type="text" placeholder="e.g Alex Smith" />
+      <FormField label="Email" type="email" placeholder="e.g. alexsmith@gmail.com" />
+    
+      <CoolButton/> */}
+      <SignUp/>
+
```
Comment (In-S4n3) [inline src/App.js:18]: Now , you can try to do the last iteration! 🗡️

File: src/coolButton/coolButton.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import 'bulma/css/bulma.css';
+
+export default function CoolButton(props) {
```
[the remark this answers is not in the document]
Comment (ZuzanaAK) [reply src/coolButton/coolButton.js:4]: Thanks! I tried both ways and forgot to delete one...

File: src/App.js [hunk 6]
```diff
@@ -1,7 +1,23 @@
...
 const App = () => {
-  return <h1>Bulma</h1>;
+  return (
+    <React.Fragment>
+      {/* <Navbar /> 
+    
+      <FormField label="Name" type="text" placeholder="e.g Alex Smith" />
+      <FormField label="Email" type="email" placeholder="e.g. alexsmith@gmail.com" />
+    
+      <CoolButton/> */}
```
[the remark this answers is not in the document]
Comment (ZuzanaAK) [reply src/App.js:16]: Thanks!

File: src/signup/Signup.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+   return (
+    <div>
+        <Navbar/>
+        <div className="signup-bloc">
+            <h1>Please, sign up here</h1>
+
+            <FormField label="Name" type="text" placeholder="e.g Alex Smith"/>
+            <FormField label="Email" type="email" placeholder="e.g. alexsmith@gmail.com"/>
+            <FormField label="Password" type="Password" placeholder="enter your password here"/>
+            <CoolButton type="submit" name = "Submit" isSmall isSuccess></CoolButton>
```
[the remark this answers is not in the document]
Comment (ZuzanaAK) [reply src/signup/Signup.js:17]: Thanks!

File: src/coolButton/coolButton.js [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    return (
+        <div>
+            <button className={className}>{props.name}</button>
+        </div>
+    )
+}
+
+
```
[the remark this answers is not in the document]
Comment (ZuzanaAK) [reply src/coolButton/coolButton.js:23]: if I put there {props.children} it doesn't work, it does work with {props.name} ...?

File: src/navbar/Navbar.js (hunk 1 above)
Comment (ZuzanaAK) [reply src/navbar/Navbar.js:6]: thanks!

File: src/App.js (hunk 3 above)
Comment (ZuzanaAK) [reply src/App.js:12]: if I uncomment it I have the navbar twice as I am passing it all into the FormFiled and CoolButton into the Signup

**Pull request closed without merging**